edbdeploy/project.py

```diff
@@ -1764,41 +1790,64 @@ def _p(s):
     @staticmethod
     def list(cloud):
         projects_path = os.path.join(Project.projects_root_path, cloud)
-        headers = ["Name", "Path", "Machines", "Resources", "Components"]
+        headers = ["Name", "Path", "Machines", "Resources", "Instance Names"]
```

Comment:
This change will be effective for all the cloud vendors. The `Components` column shows project's Ansible state which can be `UNKNOWN`, `DEPLOYED` or `FAILED`. I agree that `Components` name is not very meaningful, maybe we could rename this column to `Ansible state`. What do you think?